bug_monitor_list_view_inc.php

```diff
@@ -110,6 +110,16 @@
 		<?php } ?>
 	</td>
 </tr>
+<?php if ( ( ON === config_get_global( 'public_urls_enabled' ) ) && isset( $t_share_link ) && ( !is_null( $t_share_link ) ) && ( $t_share_link != '' ) ) { ?>	
```

Comment:
@vboctor Sorry, can you clarify? I do not see "monitor" section on printed issue, therefore there is no public link. Besides, exporting or printing also shows private notes, etc., so why would you want to hide URL?